.github/workflows/performance.yml

```diff
@@ -0,0 +1,27 @@
+name: Performance tests
+
+on:
```

Comment:
We might want to run benchmarks for every push to the main branch, so we have a reference point

Reply:
changed